apps/permissions.py

```diff
@@ -0,0 +1,47 @@
+"""
+Copyright 2019 ShipChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from django.db.models import Q
+from rest_framework import permissions
+
+
+def get_user(request):
+    if request.user.is_authenticated:
+        return request.user.id, request.user.token.get('organization_id', None)
+    return None, None
+
+
+def is_owner_q(request):
```

Comment:
maybe change the name to be a bit more descriptive of what it is actually querying against

Reply:
What would you propose? I struggled with naming this function. It contains the same logic as the is_owner function except returns `Q` filters for querysets.